CONTRIBUTING.md

```diff
@@ -1,197 +1,221 @@
+[project boards](https://github.com/rapidsai/cudf/projects).
 
-> **Pro Tip:** Always look at the release board with the highest number for
-issues to work on. This is where RAPIDS developers also focus their efforts.
+**Note:** Always look at the release board that is
+[currently under development](https://docs.rapids.ai/maintainers) for issues to work on. This is
+where RAPIDS developers also focus their efforts.
 
-Look at the unassigned issues, and find an issue to which you are comfortable
-contributing. Start with _Step 3_ above, commenting on the issue to let
-others know you are working on it. If you have any questions related to the
-implementation of the issue, ask them in the issue instead of the PR.
-
-## Setting Up Your Build Environment
-
-The following instructions are for developers and contributors to cuDF OSS development. These instructions are tested on Linux Ubuntu 16.04 & 18.04. Use these instructions to build cuDF from source and contribute to its development.  Other operating systems may be compatible, but are not currently tested.
+Look at the unassigned issues, and find an issue to which you are comfortable contributing. Start
+with _Step 3_ above, commenting on the issue to let others know you are working on it. If you have
+any questions related to the implementation of the issue, ask them in the issue instead of the PR.
 
+## Setting up your build environment
 
+The following instructions are for developers and contributors to cuDF development. These
+instructions are tested on Ubuntu Linux LTS releases. Use these instructions to build cuDF from
+source and contribute to its development. Other operating systems may be compatible, but are not
+currently tested.
 
 ### General requirements
 
 Compilers:
 
-* `gcc`     version 9.3+
-* `nvcc`    version 11.5+
-* `cmake`   version 3.20.1+
+* `gcc` version 9.3+
+* `nvcc` version 11.5+
+* `cmake` version 3.20.1+
 
 CUDA/GPU:
 
 * CUDA 11.5+
 * NVIDIA driver 450.80.02+
 * Pascal architecture or better
 
-You can obtain CUDA from [https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
+You can obtain CUDA from
+[https://developer.nvidia.com/cuda-downloads](https://developer.nvidia.com/cuda-downloads).
+
+### Create the build environment
 
-### Create the build Environment
+- Clone the repository:
 
-- Clone the repository and submodules
 ```bash
 CUDF_HOME=$(pwd)/cudf
 git clone https://github.com/rapidsai/cudf.git $CUDF_HOME
 cd $CUDF_HOME
-git submodule update --init --remote --recursive
 ```
+
 - Create the conda development environment `cudf_dev`:
+
 ```bash
 # create the conda environment (assuming in base `cudf` directory)
-# note: RAPIDS currently doesn't support `channel_priority: strict`; use `channel_priority: flexible` instead
+# note: RAPIDS currently doesn't support `channel_priority: strict`;
+# use `channel_priority: flexible` instead
 conda env create --name cudf_dev --file conda/environments/cudf_dev_cuda11.5.yml
```

Comment:
I'd also consider noting that this file is updated fairly often. It's easy for it to fall out of sync since we upgrade things like arrow fairly often and can cause somewhat mystifying build errors if you don't already know it could be a problem.

Reply:
Do you like the change in 5c967782cf?